### PR TITLE
TOMEE-2871 - Translate to Portuguese: examples/mp-rest-client

### PR DESCRIPTION
**Pull request by Daniel-Dos**
link to jira: https://issues.apache.org/jira/browse/TOMEE-2871

### CODE REVIEW

**Comment by cesarhernandezgt** [comment]
+1 in format, thank you @Daniel-Dos
Just missing now a Portuguese review.

**Comment by cassunde** [comment]
hi @Daniel-Dos, the "Run the tests for different scenarios related with JWT validation" part, should it be translated?

**Comment by Daniel-Dos** [comment]
hello @cassunde the example does not contain JWT validation input. I removed and forgot to take out English and Spanish

**Comment by Daniel-Dos** [comment]
thanks @cesarhernandezgt and @cassunde  to review

File: examples/mp-rest-client/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,150 @@
+= Cliente Rest MicroProfile
+:index-group: MicroProfile
+:jbake-type: page
+:jbake-status: published
+
+Este é um exemplo básico sobre como configurar e utilizar o Cliente Rest do MicroProfile no Tomee.
```
Comment (willesreis) [inline examples/mp-rest-client/README_pt.adoc:6]:
```suggestion
Este é um exemplo básico sobre como configurar e utilizar o Cliente Rest do MicroProfile no TomEE.
```

File: examples/mp-rest-client/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        assertTrue(bookResourceClient.getListOfBooks().size() == 1);
+        assertTrue(bookResourceClient.getBook(2).getName().equalsIgnoreCase("Top 3 Tomee Configuraiton Tips"));
+    }
+
+}
+----
+
+=== Sobre a arquitetura de teste
+
+Os casos de teste para este projeto são criados usando o Arquillian e o TomEE Remote. A configuração Arquillian pode ser encontrada em `src/test/resources/arquillian.xml`.
```
Comment (willesreis) [inline examples/mp-rest-client/README_pt.adoc:150]:
```suggestion
Os casos de teste para este projeto são criados usando o Arquillian e o TomEE Remote. A configuração do Arquillian pode ser encontrada em `src/test/resources/arquillian.xml`.
```

**Comment by Daniel-Dos** [comment]
thanks @cesarhernandezgt and @willesreis  for review .

**Comment by Daniel-Dos** [comment]
thanks .

**Pull request merged**